### ISSUE TITLE
[BITV] Full date can be shown after headings like "Today" as a hidden-visually span

### ISSUE THREAD

**Issue by JuliaKirschenheuter** [nextcloud/activity#1019]

**Comment by JuliaKirschenheuter** [comment]
Currently headings are inside of `<span>` which is inside of `<h2>`. Custom tooltip with full data is inside of an attr `data-original-title`. Since custom tooltips will be replaced with native tooltips:
1. full data will be moved to `title` attr.
2. full data will be placed in `aria-label` inside of `<h2>`
3. `<span>` isn't needed and will be removed.

All this work will be done in https://github.com/nextcloud/activity/issues/1045

Closing a "duplicate".

**Comment by JuliaKirschenheuter** [comment]
Hello @michaelnissenbaum,

could you please tell me: is it correct to place a full data in `aria-label` attr and at the same time place relative date (7 min ago) inside of a `<span>`?

![Screenshot from 2023-01-26 14-35-08](https://user-images.githubusercontent.com/6078378/214849604-385335dd-06ba-4900-817b-966899247660.png)

![Screenshot from 2023-01-26 14-35-42](https://user-images.githubusercontent.com/6078378/214849630-c4775cc0-bb4a-43b2-ba73-3e0de47f3fb1.png)


Or should be instead of it an exact visually-hidden `<span>`? Thank you!

**Comment by michaelnissenbaum** [comment]
The aria-label attribute is sometimes ignored by some assistive technology (screen reader) if the element you put it on doesn't have any semantic meaning. And in your example you use a span element which doesn't have semantic meaning. See https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-label for more details.
In your case my proposal will be as followed:

<span class="sr-only">January 26, 2023, 2:28 PM</span>
<span>7 minutes ago</span>

Class sr-only just visually hide the text making it only available for screen reader users. See https://www.tpgi.com/the-anatomy-of-visually-hidden/.
